tools/report.sh

```diff
@@ -25,7 +26,7 @@ fi
 
 
 usage() {
-	echo "Usage: $0 --namespace=<string> --cluster=<string>" 1>&2;
+	echo "Usage: $0 --namespace=<string> --cluster=<string> --secrets=(off|hidden|all)" 1>&2;
```

Comment:
Probably should mention what the default is. And does it really need to be `all`? It seems like hidden might be better since most problems wouldn't require anyone to know the secrets.